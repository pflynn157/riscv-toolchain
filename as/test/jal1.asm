jal x12, LABEL
jal x11, LABEL
LABEL:
