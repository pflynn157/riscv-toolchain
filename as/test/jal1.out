0080066f
004005ef
